### PR TITLE
Adding api key for the request

### LINKED ISSUE #137: Adding X-API-KEY for api request
Our API for dashboard is using API Key to auth the request, please add the API Key Header in global request handler. Just like [1] for reference. Please note, this key should be added by `config` file, such as `.env` file. I recommend fetching this key by `fetchAPIKey` API.

[1] https://github.com/apache/incubator-apisix-dashboard/commit/b3b306550e8e5153d42d943d2d652914e2ad88e3#diff-084c3d9c2786b7cd963be84e40a38725R32

### PR DESCRIPTION
**Pull request by sshniro**
Fix #137

### CODE REVIEW

File: src/utils/request.ts [hunk 1]
```diff
@@ -51,6 +56,9 @@ const errorHandler = (error: { response: Response }): Response => {
 const request = extend({
   errorHandler, // 默认错误处理
   credentials: 'include', // 默认请求是否带上cookie
+  headers: {
+    'X-API-KEY': fetchAPIKey()
```
Comment (juzhiyuan) [inline src/utils/request.ts]: emm `IConfig` is just a typing, we could use:

```js
import {API_KEY} from '../../config/config'

headers: {
    'X-API-KEY': API_KEY
}
```

directly here.

Comment (juzhiyuan) [reply src/utils/request.ts]: and because the APISIX doesn't support fetching APIKEY , so we could use the fixed value now.

Comment (juzhiyuan) [reply src/utils/request.ts]: sorry, there should be

image

We just exported the API_KEY standalone, do not need put it in the default config value.

Comment (sshniro) [reply src/utils/request.ts]: Thank you for the detailed explanation!

**Comment by juzhiyuan** [comment]
@sshniro Thank you for your contribution!

BTW, we'd better use `resolve #137` instead of `fix` :)

**Comment by sshniro** [comment]
Sure! @juzhiyuan

**Pull request merged**